configure.ac

```diff
@@ -10,7 +10,8 @@ AC_INIT([tpm2-tss],
         [https://github.com/tpm2-software/tpm2-tss])
 AC_CONFIG_MACRO_DIR([m4])
 : ${CFLAGS=""}
-AC_PROG_CC
+AC_PROG_CXX([clang++ g++])
```

Comment:
Are we sure that we are not killing other compilers by this ?

Reply:
My understanding is that this is an order of preference search list, aka it will try to grab clang before gcc if both are present on a system.

Reply:
Understood. My question is about CCs that are neither clang or gcc. Will they be precluded this way ?

Reply:
I don't think so. But I've gone ahead and reverted to the pervious way of no args because then CC must be given to configure which is better I think because it preserves original behavior and is explicit.